frontend/packages/ceph-storage-plugin/src/components/dashboards/block-pool/states.tsx

```diff
@@ -7,6 +7,18 @@ import {
   GrayUnknownIcon,
 } from '@console/shared/src/components/status/icons';
 
+import { ImageStates } from '../../../types';
+
+export const ImageStateLegendMap: { [state in ImageStates]: string } = {
+  [ImageStates.STARTING_REPLAY]: 'Start replay',
+  [ImageStates.STOPPING_REPLAY]: 'Stop reply',
+  [ImageStates.REPLAYING]: 'Replaying',
+  [ImageStates.STOPPED]: 'Stopped',
+  [ImageStates.ERROR]: 'Error',
+  [ImageStates.SYNCING]: 'Syncing',
+  [ImageStates.UNKNOWN]: 'Unknown',
+};
+
```

Comment:
externalize these as well
```suggestion
export const ImageStateLegendMap: { [state in ImageStates]: string } = (t) => {
  [ImageStates.STARTING_REPLAY]: t('Start replay'),
  [ImageStates.STOPPING_REPLAY]: 'Stop reply',
  [ImageStates.REPLAYING]: 'Replaying',
  [ImageStates.STOPPED]: 'Stopped',
  [ImageStates.ERROR]: 'Error',
  [ImageStates.SYNCING]: 'Syncing',
  [ImageStates.UNKNOWN]: 'Unknown',
};

```

Reply:
ack